README.md

```diff
@@ -37,30 +37,38 @@ Source: [Temporary installation in Firefox](https://developer.mozilla.org/en-US/
 
 ## Running the tests
 
-In development, it's easiest to run:
+You'll need jasmine-browser, which you can set up with:
 
 ```
-$ bundle exec rake jasmine
+$ npm test
```

Comment:
I think this needs to be updated to `npm install`

Reply:
Done!